spec/features/events_spec.rb

```diff
@@ -152,7 +152,8 @@
 
   scenario "logged in as Organizer I can not change application status with radio buttons if the applications are locked" do
     login(:organizer)
-    @event.lock_application_status
+    @event.acceptances_have_been_sent = true
+    @event.rejections_have_been_sent = true
```

Comment:
same as above, with event in execution phase?
Also, you should propably test that sending either acceptances or rejections alone is enough to lock the participant selection.

Also the test in line 162 (cant mark it) could be updated, since we check for labels now (afaik) when we want to test for radio buttons, not for normal text. It's not that important tho